test/data_check/check_rotation.py

```diff
@@ -69,29 +67,17 @@
 tcorr[1] = sinb * crap[7] - cosb * crap[6]
 tcorr[2] = crap[8]
 tcorr[3] = (
-    -cosa * cosb * crap[4]
```

Comment:
Here's a case where this formatting may make more sense than what flake8 suggests. I.e. the code is probably more readable as multiple lines. Can we add an ignore for the rule here?

@mdenolle wdyt?